src/AutoBuild.jl

```diff
@@ -815,7 +817,7 @@ function autobuild(dir::AbstractString,
         compress_dir(joinpath(dest_prefix.path, "logs"), verbose=verbose)
 
         # Once we're built up, go ahead and package this dest_prefix out
-        tarball_path, tarball_hash, git_hash = package(
+        tarball_path, tarball_hash, git_hash = _package_fast(
```

Comment:
Ugh, this is a different function named `package`...

Reply:
This is calling the `BinaryBuilderBase.package` function which should call the the appropriate `_package_fast`